internal/eval/results.go

```diff
@@ -0,0 +1,14 @@
+package eval
+
+type RespType int
+
+const (
```

Comment:
Shall we add a comment to not change the order and be careful when adding items here?

Reply:
Yes, lets do that...

Reply:
Done.